src/js/views/admin/shared/clinician-groups.scss

```diff
@@ -0,0 +1,55 @@
+.clinician-groups {
+  .clinician-groups__item:first-child {
+    border-radius: 4px 4px 0 0;
+  }
+
+  &.is-disabled {
+    cursor: not-allowed;
+    opacity: 0.5;
+  }
+}
+
+.clinician-groups__item {
+  border: 1px solid $black-90;
+  border-bottom-width: 0;
+  border-radius: 0;
+  display: block;
+  font-size: 14px;
+  font-weight: 600;
+  line-height: 16px;
+  overflow: hidden;
+  padding: 8px 16px 8px 40px;
+  position: relative;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+.clinician-groups__item-icon .icon {
+  font-size: 16px;
+  left: 14px;
```

Comment:
This is because the Fontawesome "users" icon is 4px wider than the "user-circle" icon. So the button text does in the Role and Groups components does not align properly without that 14px definition. Is there a better way to account for this? I was basing this off of the Marvel https://marvelapp.com/7ihgbgb/screen/63561658/handoff

Reply:
elsewhere we've done
```css
  // Handles odd width icon spacing
  margin-right: -2px;
```
At least there should be a comments as to why it's not 4x, but if we can handle it the same way that's probably ideal.

Reply:
Ah perfect, thanks.